WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletViewModel.cs

```diff
@@ -82,7 +82,7 @@ public RecoverWalletViewModel(WalletManagerViewModel owner) : base("Recover Wall
 						var km = KeyManager.Recover(mnemonic, Password, filePath: null, keyPath, MinGapLimit);
 						km.SetNetwork(Global.Network);
 						km.SetFilePath(walletFilePath);
-						km.ToFile();
+						Global.WalletManager.AddWallet(km);
```

Comment:
nit: use a WalletManager property instead of global...

Reply:
ACK